app/server.js

```diff
@@ -1,12 +1,12 @@
 "use strict";
 
-const http = require("http"),
+const https = require("https"),
   express = require("express");
 
 const ServerFactory = function(config, logger) {
 
   const app = express(),
-    server =  http.createServer(app);
```

Comment:
@rodrigopavezi In previous investigation and research on this I'd come across this - https://goo.gl/XWleJ4 . Scroll down to "If you must listen on same port", solution proxies incoming connection and redirects respectively. Not sure how reliable of a solution this is, but it looks good. Thoughts?